test/run_cli.js

```diff
@@ -0,0 +1,41 @@
+'use strict';
+
+const ChildProcess = require('child_process');
+const Path = require('path');
+
+const labPath = Path.join(__dirname, '..', 'bin', '_lab');
+
+module.exports = (args, callback) => {
+
+    return new Promise((resolve, reject) => {
+
+        const cli = ChildProcess.spawn('node', [labPath, ...args]);
```

Comment:
@Marsup It works in node 5 without a flag (tested on `v5.5.0`) as it’s not the "rest operator" but the "spread operator". (I find it pretty confusing that they look the same, have similar semantics but have completely different platform support.)